src/components/Notification/types.ts

```diff
@@ -0,0 +1,29 @@
+import { iconTypes } from "../../components/Icon/collection";
+
+export interface NotificationProps {
+        /**
+         * The notification ID will generated if not assigned
+         */
+        id?: string;
+    
+        /**
+         * The title to display in notification
+         */
+        title?: string
+    
+        /**
+         * The message to display in notification
+         */
+        message?: string;
```

Comment:
i think `message` should probably not be optional, i cant think of a case where we would nag a user with a notification that had no text. The Kresmir message is great but lets keep that as a Storybook easter egg so its not hard baked into our component

Reply:
Gotch ya!